src/component/Header/index.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import styled from "styled-components";
+import { BiCamera } from "react-icons/bi";
+import TextBox from "../TextBox";
+import { AiOutlineMenu } from "react-icons/ai";
+const HeaderWrapper = styled.div`
+    widht: 100%;
+    height: 56px;
+    background-color: #343a40;
+    display: flex;
+
+    justify-content: center;
+    // 주축을 중심으로 정렬
+
+    align-items: center;
+    // 보조축을 중심으로 정렬
+`;
```

Comment:
```suggestion

    justify-content: center;
    /* 주축을 중심으로 정렬 */

    align-items: center;
    /* 보조축을 중심으로 정렬 */
`;
```

CSS는 `//` 를 주석으로 인식하지 못해요. `/* ~ */` 로 변경해볼까요?

Reply:
<img width="242" alt="image" src="https://user-images.githubusercontent.com/64801796/168457407-cc7fc984-71b5-4657-90f1-7aa41f72b14b.png">
수정 완료 했습니다!